src/pages/components/Navbar/DeveloperNavbar.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import { Dialog } from '@headlessui/react';
+import { Bars3Icon, XMarkIcon } from '@heroicons/react/24/outline';
```

Comment:
we don't need these packages since we have icons from material UI or chakra - please uninstall these packages